lib/hammer_cli_csv/host_groups.rb

```diff
@@ -45,66 +47,77 @@ def export
             realm = hostgroup['realm_name']
             puppet_proxy = hostgroup['puppet_proxy_id'] ? foreman_host(:id => hostgroup['puppet_proxy_id']) : nil
             puppet_ca_proxy = hostgroup['puppet_ca_proxy_id'] ? foreman_host(:id => hostgroup['puppet_ca_proxy_id']) : nil
-            # TODO: http://projects.theforeman.org/issues/7597
-            # content_source = hostgroup['content_source_id'] ? foreman_host(:id => hostgroup['content_source_id']) : nil
-            content_source = nil
+            content_source = hostgroup['content_source_id'] ? foreman_host(:id => hostgroup['content_source_id']) : nil
```

Comment:
Good catch, thanks! I added a needinfo to the redmine. The issue no longer exists for my usage of the API here so the TODO is no longer relevant.